docs/src/examples/do-events.jl

```diff
@@ -0,0 +1,244 @@
+
+!!! info "Future improvement"
+    Supporting the computations for uneven timeseries is a planned improvement of TransitionsInTimeseries. This will avoid the need of regridding data on coarse grids and will prevent from introducing any bias.
+
+## Hindcasting simulated DO-events
+
+In CLIMBER-X, the EMIC described in [willeit-earth-2022](@cite), DO-like events can be triggered by forcing the North Atlantic with a (white noise) freshwater input. Simulated DO-like events present the big advantage of being evenly sampled in time and free of measurement noise. We run this analysis over two exemplary simulation outputs:
+=#
+
+t_transitions = [[1, 1850, 2970, 3970, 5070, 5810, 7050, 8050],
+                 [1, 3500, 4370, 5790, 7200, 8140]]
+t_lb = [[300, 500, 300, 600, 300, 500, 500], [1800, 500, 1000, 900, 500]]
+tseg_start = [t_transitions[1][1:end-1] + t_lb[1], t_transitions[2][1:end-1] + t_lb[2]]
+tseg_end = [t_transitions[1][2:end] .- 50, t_transitions[2][2:end] .- 50]
+
+figvec = Figure[]
+
+for j in 1:2
+    ## Download the data and perform loess filtering on it
+    tmp = Base.download("https://raw.githubusercontent.com/JuliaDynamics/JuliaDynamics/" *
+        "master/timeseries/climberx-do$(j)-omaxa.csv")
+    data = readdlm(tmp)
+    tcx, xcx = data[1, 1000:end], data[2, 1000:end]
+    t, x, xtrend, r = loess_filter(tcx, xcx, span = 0.02)
+
+    ## Initialize figure
+    xlims = (0, last(tcx))
+    xticks = kyr_xticks(xlims[1]:1e3:xlims[2])
+    fig, axs = plot_do(tcx, xcx,  t, xtrend, t, r, t_transitions[j], extrema(t), xticks)
+    ylims!(axs[1], (5, 40))
+    axs[1].ylabel = L"Max. Atlantic overturning (Sv) $\,$"
+
+    ## Run sliding analysis and update figure with results
+    dt = mean(diff(tcx))
+    config = SegmentWindowConfig(
+        indicators, change_metrics, tseg_start[j], tseg_end[j],
+        whichtime = last, width_ind = Int(200÷dt), min_width_cha = 50)
+    results = estimate_indicator_changes(config, r, t)
+    signif = SurrogatesSignificance(n = 1_000, tail = :right, rng = Xoshiro(1995))
+    flags = significant_transitions(results, signif)
+
+    plot_segment_analysis!(axs, results, signif)
+    vlines!(axs[1], t_transitions[j], color = Cycled(1), linewidth = 3)
+    push!(figvec, fig)
+end
+figvec[1]
+
+#=
+It here appears that not all transitions are preceeded by a significant increase of variance and AC1, even in the case of clean and evenly sampled time series. Let's check another case:
+=#
+
+figvec[2]
+
+#=
+Same here! Although CLIMBER-X does not represent real DO-events, the above-performed analysis might be hinting at the fact that not all DO transitions can be forecasted with CSD. Nonetheless, performing a CSD analysis can inform on the evolution of a system's resilience.
+
+# References
```

Comment:
Move the references to a dedicated new top level page called "References" (like we have a tpp level page for Tutorial)